index.bs

```diff
@@ -236,7 +236,7 @@ When asked to <dfn abstract-op>remove client hints from redirect if needed</dfn>
 
 <ol>
  <li>If |request|'s [=client=] is null, then abort these steps.
- <li>Let |clientHintsSet| be |request|'s <var ignore>client</var>'s [=environment settings object/client hints set=].
+ <li>Let |clientHintsSet| be the set of all client hints.
```

Comment:
Is it useful to say "...all client hints supported by the [=user agent=]" or "all supported client hints"? 

I guess it's obvious you can't filter out hints you haven't implemented, but when I read this I want to click on "all client hints"... adding "supported" sort of hand waves that away.